src/stores/pacbioLibraryBatchCreate.js

```diff
@@ -86,6 +87,25 @@ export const usePacbioLibraryBatchCreateStore = defineStore('pacbioLibraryBatchC
       try {
         const csv = await csvFile.text()
         const sources = getColumnValues(csv, 0)
+        const tagNames = getColumnValues(csv, 1)
+
+        // Check for duplicate sources
+        let duplicates = findDuplicates(sources)
```

Comment:
Instead of just checking the array length with a Set instead of findDuplicates - I thought  when there are a lot of records, if It returns the duplicated values, makes it easier for users to spot and fix mistakes.